core/app/[locale]/not-found.tsx

```diff
@@ -66,30 +63,16 @@ export default async function NotFound() {
           <h2 className="text-4xl font-black lg:text-5xl">{t('heading')}</h2>
           <p className="text-lg">{t('message')}</p>
         </div>
-        <NextIntlClientProvider locale={locale} messages={{ NotFound: messages.NotFound ?? {} }}>
-          <SearchForm />
-        </NextIntlClientProvider>
-        {featuredProducts.length ? (
-          <section>
-            <h3 className="mb-8 text-3xl font-black lg:text-4xl">{t('featuredProducts')}</h3>
-            <div className="grid grid-cols-2 gap-x-8 gap-y-8 md:grid-cols-4">
-              {featuredProducts.map((product) => (
-                <NextIntlClientProvider
-                  key={product.entityId}
-                  locale={locale}
-                  messages={{ Product: messages.Product ?? {} }}
-                >
-                  <ProductCard product={product} showCart={false} showCompare={false} />
-                </NextIntlClientProvider>
-              ))}
-            </div>
-          </section>
-        ) : null}
+        <SearchForm />
+        <ProductCardCarousel
+          products={featuredProducts}
+          showCart={false}
+          showCompare={false}
+          title={t('Carousel.featuredProducts')}
+        />
       </main>
 
       <Footer data={data.site} />
     </>
   );
 }
-
-export const runtime = 'edge';
```

Comment:
Is there a reason we are removing the runtime from this page?

Reply:
Mistake, thanks for catching it!